patches/components-permissions-android-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
+index 3cfa933430a16ea55bfae73aa02e195a0aedd5a9..79d9bfcd28fa8b1c490213cb34abd31943a64e88 100644
+--- a/components/permissions/android/BUILD.gn
++++ b/components/permissions/android/BUILD.gn
+@@ -14,6 +14,7 @@ generate_jni("jni_headers") {
+     "java/src/org/chromium/components/permissions/PermissionUtil.java",
+     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
+   ]
++  sources += brave_components_permissions_java_jni_sources
+ }
+ 
+ android_resources("java_resources") {
+@@ -124,6 +125,7 @@ android_library("java") {
+     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelPrompt.java",
+     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
+   ]
++  sources += brave_components_permissions_java_sources
```

Comment:
@bridiver
As for eliminating this line wiht `brave_components_permissions_java_sources` in patch, I recall I had errors like `class XXX not found` when my new java classes were in the different jar than Chromium's classes I tried to use. 
So maybe it is possible, but will require even more patches than that single line.